packages/strapi-plugin-i18n/controllers/content-types.js

```diff
@@ -50,7 +53,9 @@ module.exports = {
 
     ctx.body = {
       nonLocalizedFields: sanitizedNonLocalizedFields,
-      localizations: entity.localizations.concat(pick(['id', 'locale'], entity)),
+      localizations: entity.localizations.concat(
```

Comment:
Any reason not to return the localizations directly as they only contain the properties you are selecting anyway ?

Reply:
There is a `concat` actually, here all the locales are set not only "other locales"